_build/templates/default/package.json

```diff
@@ -1,31 +1,40 @@
 {
-  "name": "revolution-theme-default",
-  "title": "MODX Revolution",
-  "version": "3.0.0",
-  "license": "GPL-2.0+",
-  "repository": "https://github.com/modxcms/revolution",
-  "devDependencies": {
-    "grunt": "^1.3.0",
-    "grunt-autoprefixer": "^3.0.4",
-    "grunt-contrib-concat": "^1.0.1",
-    "grunt-contrib-copy": "^1.0.0",
-    "grunt-contrib-csslint": "^2.0.0",
-    "grunt-contrib-cssmin": "^3.0.0",
-    "grunt-contrib-uglify": "^4.0.1",
-    "grunt-contrib-watch": "^1.1.0",
-    "grunt-imageoptim": "^1.4.4",
-    "grunt-notify": "^0.4.5",
-    "grunt-sass": "^3.1.0",
-    "node": "^14.15.0",
-    "node-sass": "^5.0.0"
-  },
-  "dependencies": {
-    "bourbon": "^5.1.0",
-    "bourbon-neat": "^3.0.0",
-    "@fortawesome/fontawesome-free": "^5.7.0",
-    "normalize-scss": "^7.0.1"
-  },
-  "scripts": {
-    "build": "grunt build"
-  }
+    "name": "revolution-theme-default",
+    "title": "MODX Revolution",
+    "version": "3.0.0",
+    "license": "GPL-2.0+",
+    "repository": "https://github.com/modxcms/revolution",
+    "devDependencies": {
+        "@lodder/grunt-postcss": "^3.0.1",
+        "autoprefixer": "^10.3.3",
+        "bourbon": "^5.1.0",
+        "bourbon-neat": "^3.0.0",
+        "cssnano": "^5.0.8",
+        "grunt": "^1.4.1",
+        "grunt-contrib-concat": "^1.0.1",
+        "grunt-contrib-copy": "^1.0.0",
+        "grunt-contrib-watch": "^1.1.0",
+        "grunt-dart-sass": "^2.0.1",
+        "grunt-notify": "^0.4.5",
+        "grunt-terser": "^2.0.0",
+        "node": "^14.15.0",
```

Comment:
windows 7 is not supported by node.js 14 :( Windows 7 still be more popular than 8, at least in Russia.
Is this dependency is necessary?

Reply:
Windows 7 as you probably know has been unsupported by Microsoft for more that a year and a half now. That said, I've seen that people on 7 have been able to get node 14+ running. See https://github.com/nodejs/node/issues/33000#issuecomment-644530517

Reply:
Thanks for link, I'll try it. 
Anyway, I would ask again: for what purpose is this dependency?

Reply:
It's the base software needed to use npm and the modules it manages. Are you intending on doing some development on the manager interface? Note that this same dependency has existed in 2.x for quite a while now.

Reply:
It's very strange to have the npm installed and not have the nodejs installed, isn't it?

Reply:
@mishantrop yes it is. I assume you're running Windows 7?

Reply:
Yes, including

Reply:
Also there are better ways to limit the version of the application:

`package.json`

```json
{
  "name": "modx",
  "version": "3.0.0",
  "engines": {
   "node": ">=12.0.0"
  }
}
```

`.npmrc`
```config
engine-strict=true
```

I have not personally tested it as it was never necessary.

Source: https://stackoverflow.com/a/68653111